spec/libsass-closed-issues/issue_2569/error-dart-sass

```diff
@@ -1,6 +1,6 @@
 Error: expected "@".
```

Comment:
Probably need another misleading error message for the `expected "@"` too if you don't already.

Reply:
Good catch. https://github.com/sass/dart-sass/issues/584